### ISSUE TITLE
Write BivariateIndicator class

### ISSUE THREAD

**Issue by huard** [Ouranosinc/xclim#98]
### Description

Same thing as UnivariateIndicator, but for two variables...

**Comment by Zeitsperre** [comment]
Just curious, but would this simply be an extension to the UnivariateIndicator class? It seems to me that almost all of the criteria are the same.

**Comment by huard** [comment]
Correct. In fact I'll see if I can write a generic Indicator class that would handle any number of variable while remaining readable.
